frontend/components/atoms/AUserName.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+import { Box } from '@material-ui/core';
+import Image from 'next/image';
+
+interface AUserNameProps {
+  username: string;
+  avatar?: string;
+  // hasAvatar: boolean;
+  // isLiked: boolean;
+  // isUsed: boolean;
+  // isListing: boolean;
+  // username: string;
+  // title: string;
+  // avatar?: string
+}
```

Comment:
Consider removing unused props or adding them as optional props (with the ? eg. `hasAvatar?: boolean`)